Tests/test_raxml_tool.py

```diff
@@ -8,9 +8,8 @@
 import os
 import unittest
 
-from Bio import Phylo
+from Bio import MissingExternalDependencyError, Phylo
```

Comment:
Not sure if this is encouraged, can change it back if you want to have only 1 import per line.

Reply:
I have the feeling that the general trend is to have one import per line. I'm often using several imports per line if they are closely related (e.g. `from urllib.request import urlopen, Request`). However, here this is not the case, thus I would put them on different lines.

Reply:
Reverted, now commit only splits line. Thanks for your input!

Reply:
Agreed - separate lines except perhaps examples like Markus' where it is several functions/classes from a module. Here you are importing an exception and a sub-module, nicer to have them on separate lines.

On this note, I quite like how ``reorder-python-imports`` reformats imports:

https://github.com/asottile/reorder_python_imports

I am somewhat tempted to suggest applying this to Biopython....